src/PreflightTest.ts

```diff
@@ -37,176 +76,545 @@ export interface PreflightTest {
     listener: PreflightTest.Listener.Connected
   ): this;
 
+  /**
+   * Completed event. Raised when the PreflightTest has successfully completed.
+   *
+   * @example
+   * ```typescript
+   * preflightTest.addListener(PreflightTest.Event.Completed, (report: string) => {
```

Comment:
I think the report is a json string. Since we're now in JS layer at this point, can we define report object instead? We have this on Voice JS for reference https://twilio.github.io/twilio-voice.js/interfaces/PreflightTest.Report.html

Reply:
please update this example. report: string -> report: PreflightTest.Report